awscli/topics/s3-config.rst

```diff
@@ -0,0 +1,135 @@
+:title: AWS CLI S3 Configuration
+:description: Advanced configuration for AWS S3 Commands
+:category: S3
+:related command: s3 cp, s3 sync, s3 mv, s3 rm
+
+The ``aws s3`` transfer commands, which include the ``cp``, ``sync``, ``mv``,
+and ``rm`` commands, have additional configuration you can use to control
+S3 transfers.  This topic guide discusses these parameters as well as best
+practices and guidelines for setting these values.
+
+Before discussing the specifics of these values, note that these values are
+entirely optional.  You should be able to use the ``aws s3`` transfer commands
+without having to configure any of these values.  These configuration values
+are provided in the case where you need to modify one of these values, either
+for performance reasons or to account for the specific environment where these
+``aws s3`` commands are being run.
+
+
+Configuration Values
+====================
+
+These are the configuration values you can set for S3:
+
+* ``max_concurrent_requests`` - The maximum number of concurrent requests.
+* ``max_queue_size`` - The maximum number of tasks in the task queue.
+* ``multipart_threshold`` - The size threshold where the CLI uses multipart
+  transfers.
+* ``multipart_chunksize`` - When using multipart transfers, this is the chunk
```

Comment:
For consistency:

The chunk size the CLI uses for multipart transfers of individual files